src/components/PostBody.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import DOMPurify from 'isomorphic-dompurify';
+
+function PostBody({ content }: { content: string }) {
+  const clean = DOMPurify.sanitize(content, {
+    // ALLOWED_TAGS: ['p', 'h1'],
+    // ALLOWED_ATTR: ['className', 'dir'],
+    // SAFE_FOR_TEMPLATES: true,
+    // USE_PROFILES: { html: true },
```

Comment:
we can go back to the docs and get them from there

I will tell you why this is not good, imagine that we didn't want to use any of these options for the next year, and after this year we came and want to use one of them, in this year the library might change, and these options might not be relevant anymore, so, in this case, we need to spend time just to understand what is this, and we will remove them to add the new updated options

this is only one scenario

comments are bad, they will get outdate as soon as we merge them into the code base, and they are ugly, they take space without providing value, and much more reasons for that, it would be cool if you research more on that